kubernetes/ibm-ppc64le/OWNERS

```diff
@@ -0,0 +1,16 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+filters:
```

Comment:
no need to use filters, lets keep a simple file with approvers, reviewers and labels.